bin/push_docker.js

```diff
@@ -30,36 +28,41 @@ const currentBranch = execSync('git rev-parse --abbrev-ref HEAD')
   .toString()
   .trim();
 
-const distributionParam = process.argv[2];
-const stageParam = process.argv[3];
-const suffix = distributionParam ? `-${distributionParam}` : '';
-const stage = stageParam ? `-${stageParam}` : '';
+const distributionParam = process.argv[2] || '';
+const stageParam = process.argv[3] || '';
 const commitSha = process.env.GITHUB_SHA || 'COMMIT_ID';
 const commitShaLength = 7;
 const commitShortSha = commitSha.substring(0, commitShaLength - 1);
-const configurationEntry = `wire-web-config-default${
-  suffix ? suffix : currentBranch === 'main' ? '-main' : '-staging'
+const configurationEntry = `wire-web-config-default-${
+  distributionParam ? distributionParam : currentBranch === 'main' ? 'main' : 'staging'
 }`;
-const dependencies = {
-  ...appConfigPkg.dependencies,
-  ...appConfigPkg.devDependencies,
-  ...appConfigPkg.peerDependencies,
-  ...appConfigPkg.optionalDependencies,
-};
 
-const configVersion = dependencies[configurationEntry].split('#')[1];
+const configVersion = appConfigPkg.dependencies[configurationEntry].split('#')[1];
```

Comment:
Does it make sense to (A) simplify it that way ([app-config as no other fields](https://github.com/wireapp/wire-account/blob/staging/app-config/package.json)), and (B) to port this also to the other two repos?

Reply:
Yes, it would make sense to also add `app-config/package.json`. Basically that requires the copyconfig tool. It was introduced after the fact and never applied to this project.

https://github.com/wireapp/wire-webapp/blob/dev/.copyconfigrc.js
https://github.com/wireapp/wire-webapp/blob/dev/package.json#L214

Reply:
I think there is a misunderstanding :)
`app-config/package.json` already exists in this repository. I was just proposing to simplify the code, because that file only contains the `dependencies` field, not all the others like `devDependencies` or `peerDependencies`. Or I'm just missing a piece of the puzzle here, like other configs being pulled in during runtime or further down some automated process.

Reply:
Ah, I see ya, sounds good to me.